api/src/test/java/org/pmiops/workbench/notebooks/NotebookUtilsTest.java

```diff
@@ -31,9 +31,8 @@ public void testAppendFileExtension_newFileNotHavingExtension() {
 
   @Test
   public void testAppendFileExtension_unsupported() {
-    // lower case of Rmd which is not a supported file format
```

Comment:
lower case is now supported

Reply:
I'd like to see uppercase .SAS, .iPyNb, etc so we can see that this is intended